### DISCUSSION TITLE
Wyszukiwanie na stronie głównej

### DISCUSSION THREAD

**Discussion by mlazowik** [po8klasie/po8klasie#77] [Ideas]
Czy przy obecnej nawigacji nie będzie tak, że >90% użytkowników będzie klikać CTA bez przeglądania tekstu poniżej?

**Comment by karoli-na** [comment]
Też mi się tak wydaje, ja bym była za przerzuceniem wyszukiwania na stronę główną. Zrobiłam taką wstępną wersję tej hybrydy - jest trochę jak na stronach typu Morizon czy Otodom - na stronie głównej wersja skrócona filtrów, więcej opcji do wyboru po przejściu na docelową stronę wyszukiwania. [https://xd.adobe.com/spec/757bb86b-c6a3-454a-5cf2-34f910f4f603-034a/screen/c5ffc429-b930-4cec-be00-bf7e5f74e195/HOME-new](url)

**Comment by micorix** [comment]
@karoli-na myślę, że powinniśmy dodać też, pomimo wersji uproszczonej wyszukiwania, input z nazwą szkoły.

**Comment by karoli-na** [comment]
@micorix  I wtedy przy wyszukiwaniu po nazwie przenosi nas od razu do profilu konkretnej szkoły czy do listy szkół? Jeśli do konkretnego profilu, to musielibyśmy wtedy mieć pewnie podpowiedzi w dropdownie, bo wątpię, żeby każdy wpisywał pełną poprawną nazwę :)

**Comment by micorix** [comment]
Chyba na teraz najprościej byłoby do listy szkół
